packages/vats/src/core/types.ts

```diff
@@ -0,0 +1,511 @@
+    | 'auctioneer'
+    | 'VaultFactory'
+    | 'VaultFactoryGovernor'
+    | 'econCommitteeCharter'
+    | 'walletFactory'
+    | 'provisionPool'
+    | 'reserve'
+    | 'reserveGovernor'
+    | 'Pegasus';
+  oracleBrand: 'USD';
+  uiConfig: 'VaultFactory';
+};
+
+type WellKnownSpaces = {
+  issuer: {
+    produce: Record<WellKnownName['issuer'], Producer<Issuer>>;
+    consume: Record<WellKnownName['issuer'], Promise<Issuer>> & {
+      BLD: Promise<Issuer<'nat'>>;
+      IST: Promise<Issuer<'nat'>>;
+    };
+  };
+  brand: {
+    produce: Record<WellKnownName['issuer'], Producer<Brand>> & {
+      timer: Producer<import('@agoric/time').TimerBrand>;
+    };
+    consume: Record<WellKnownName['issuer'], Promise<Brand>> & {
+      BLD: Promise<Brand<'nat'>>;
+      IST: Promise<Brand<'nat'>>;
+      timer: Producer<import('@agoric/time').TimerBrand>;
+    };
+  };
+  oracleBrand: {
+    produce: Record<WellKnownName['oracleBrand'], Producer<Brand>>;
+    consume: Record<WellKnownName['oracleBrand'], Promise<Brand>>;
+  };
+  installation: {
+    produce: Record<WellKnownName['installation'], Producer<Installation>>;
+    consume: Record<
+      WellKnownName['installation'],
+      Promise<Installation<unknown>>
+    > & {
+      auctioneer: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/auction/auctioneer.js').start
+        >
+      >;
+      centralSupply: Promise<
+        Installation<typeof import('@agoric/vats/src/centralSupply.js').start>
+      >;
+      committee: Promise<
+        Installation<
+          typeof import('@agoric/governance/src/committee.js')['start']
+        >
+      >;
+      contractGovernor: Promise<
+        Installation<
+          typeof import('@agoric/governance/src/contractGovernor.js')['start']
+        >
+      >;
+      econCommitteeCharter: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/econCommitteeCharter.js')['start']
+        >
+      >;
+      feeDistributor: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/feeDistributor.js').start
+        >
+      >;
+      mintHolder: Promise<
+        Installation<typeof import('@agoric/vats/src/mintHolder.js').start>
+      >;
+      psm: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/psm/psm.js')['start']
+        >
+      >;
+      provisionPool: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/provisionPool.js')['start']
+        >
+      >;
+      reserve: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/reserve/assetReserve.js')['start']
+        >
+      >;
+      VaultFactory: Promise<
+        Installation<
+          typeof import('@agoric/inter-protocol/src/vaultFactory/vaultFactory.js')['start']
+        >
+      >;
+      walletFactory: Promise<
+        Installation<import('@agoric/smart-wallet/src/walletFactory.js').start>
```

Comment:
Is this a typo, or is `typeof` required here? The cases above needed it.
```suggestion
        Installation<typeof import('@agoric/smart-wallet/src/walletFactory.js').start>
```

Reply:
yes. .ts syntax is less forgiving that .js/JSDoc syntax in this regard